### PR TITLE
Add a new onedata_failsafe option

### PR DESCRIPTION
**Pull request by enolfc**
So the datahub does not fail when starting

# Summary

Avoid failing whenever datahub is not available. 

---

**Related issue :** #75

### CODE REVIEW

**Comment by andrea-manzi** [comment]
@enolfc  if i understood correctly  this change, in case we return an empty token instead of throwing an exception, the oneclient mount will fail on the sidecar container but this won't prevent the notebook container to start , correct?

**Comment by enolfc** [comment]
yes, that's the initial idea. The sidecar should not even be created as the token will not be there

File: egi_notebooks_hub/onedata.py [hunk 1]
```diff
@@ -44,6 +44,10 @@ class OnedataAuthenticator(EGICheckinAuthenticator):
 
     map_users = Bool(False, config=True, help="""perform mapping""")
 
+    onedata_failsafe = Bool(
+        True, config=True, help="""do not fail of onedata is not responsive"""
```
Comment (andrea-manzi) [inline egi_notebooks_hub/onedata.py:48]:
```suggestion
        True, config=True, help="""do not fail if onedata is not responsive"""
```

**Comment by andrea-manzi** [comment]
ok thanks i see it in the code now where the creation of the sidecar is skipped, i guess we can go for it and think about the warning to the user later

**Review by andrea-manzi — APPROVED** [review]
LGTM!

**Pull request merged**